extensions/amp-analytics/0.1/vendors.js

```diff
@@ -47,11 +47,15 @@ export const ANALYTICS_CONFIG = {
   },
 
   'googleanalytics': {
+    'vars': {
+      'eventValue': "0",
+      'documentLocation': 'AMPDOC_URL'
+    },
     'requests': {
       'host': 'https://www.google-analytics.com',
-      'basePrefix': 'v=1&_v=a0&aip=true&_s=${hitCount}&dl=${canonicalUrl}&' +
+      'basePrefix': 'v=1&_v=a0&aip=true&_s=${hitCount}&' +
```

Comment:
Thanks for catching that. Added document referrer and also added a comment about vars that would be nice to have...